docs/bis/intro/codes.md

```diff
@@ -39,25 +39,40 @@ For example, a physical valve will have its serial number permanently affixed to
 
 ## CodeSpec
 
-A `CodeSpec` (aka **Code Specification**) captures the rules for encoding and decoding significant business information into and from a Code.
-A `CodeSpec` is associated to an Element subclass via configuration of the system that edits the BIS Repository, such as the iTwin.js library, which can collaborate with a shared service (e.g. by an 'Identification Code Service') to generate and validate Codes.
-
+A `CodeSpec` (aka **Code Specification**) names and specifies a new *classification* for Codes.
+A `CodeSpec` also captures the rules for encoding and decoding significant business information into and from a Code.
+For example, the Codes for ViewDefinitions and the Codes for Equipment have different *encoding* rules and uniqueness constraints, so would each have a separate `CodeSpec`.
+
+Typically, a `CodeSpec` has a strong correlation with a branch of the Element class hierarchy and is often named after a abstract base class that defines the starting point of that branch.
```

Comment:
/s/a abstract/an abstract

Reply:
@pmconne This PR merged before I had a chance to correct. Will fix in one of my upcoming PRs.